src/odoc.ml

```diff
@@ -421,7 +421,7 @@ module Gen (S : sig val sctx : SC.t end) = struct
     |> List.sort ~compare:(fun (x, _) (y, _) ->
       Lib_name.compare (Lib.name x) (Lib.name y))
     |> List.iter ~f:(fun (lib, modules) ->
-      Printf.bprintf b "{2 Library %s}\n" (Lib_name.to_string (Lib.name lib));
+      Printf.bprintf b "{0 Library %s}\n" (Lib_name.to_string (Lib.name lib));
```

Comment:
Ah, sorry, I missed the `List.iter` above. The page needs a separate title, with `{0`, which I think Dune is not generating at all right now. Below that, all these section headings should be `{1`.

Reply:
Done. See the last commit.